apps/server/swagger/submission-api.yml

```diff
@@ -56,20 +56,35 @@
       503:
         $ref: '#/components/responses/ServiceUnavailableError'
 
-/submission/{submissionId}/{entityName}:
+/submission/{submissionId}/{actionType}:
```

Comment:
Changed this operation to receive new params `actionType,  `entityName` and `index` to delete specific items within the Active Submission